lib/config.ts

```diff
@@ -29,6 +29,9 @@ export type Config = {
     memory: {
         max: number;
     };
+    mox: {
+        cookie: string;
+    };
```

Comment:
Please add it in between
https://github.com/DIYgod/RSSHub/blob/3e91420ed54d10ef9494c0100937beb0d58cae4c/lib/config.ts#L193-L199

Reply:
thanks for your review, fixed in https://github.com/DIYgod/RSSHub/pull/15038/commits/a0d639542a25c0c9869677db93e9baa51f7ae78a